src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,11 @@
+<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee 
+         http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
+         version="3.1">
+   <listener>
+      <listener-class>
+         org.libertybikes.ai.service.AI 
```

Comment:
instead of using a web.xml here, lets use the annotation equivalent, such as this example:
https://www.codejava.net/java-ee/servlet/weblistener-annotation-examples